ansible/roles/test/tasks/check_testbed_interfaces.yml

```diff
@@ -0,0 +1,57 @@
+- block:
+  - name: Gathering lab graph facts about the device
+    conn_graph_facts: host={{ inventory_hostname }}
+    connection: local
+
+  - name: Fanout hostname
+    set_fact: fanout_switch={{ device_conn['Ethernet0']['peerdevice'] }}
+
+  - name: Check Fanout interfaces
+    local_action: shell ansible-playbook -i lab fanout.yml -l {{ fanout_switch }} --tags check_interfaces_status
+    ignore_errors: yes
```

Comment:
Where the output does go?

Reply:
All gathered info goes to the execution console, where it will be extracted if needed.

Reply:
Can we save the output into the variable and output it as
```
debug: var: output.stdout_lines
```

Reply:
Actually, it has register/debug of output:
When _interfaces.yml_ fails,
it calls _check_testbed_interfaces.yml_ with _check_fanout: true_

_check_testbed_interface.yml_  plays another playbook - _fanout.yml_ with tag check_interfaces_status:
`local_action: shell ansible-playbook -i lab fanout.yml -l {{ fanout_switch }} --tags check_interfaces_status`
which in turns calls fanout main.yml
which contains our custom block (which was not upstreamed):
```
 ###################################################################
 # Check Fanout interfaces status                                  #
 ###################################################################
- block:
  - name: Check Fanout interfaces status
    action: apswitch template=roles/fanout/templates/mlnx_interfaces_status.j2
    connection: switch
    register: fanout_interfaces
    args:
      login: "{{ switch_login['MLNX-OS'] }}"

  - debug:
      msg: "{{ fanout_interfaces.stdout.split('\n') }}"

  when: peer_hwsku == "MLNX-OS"
  tags: check_interfaces_status
```
Here we expect any team may have their own block relevant to specific vendor, called by
`  tags: check_interfaces_status`

That is why you might have not got any information when you ran the test.